### PR TITLE
Next in series: test with cutout

### PR DESCRIPTION
**Pull request by OliverJAsh**
Same as #12157, but this time we want to test if having a cut out increases the impact

image

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    };
+
+    var $articleBody = $('.content__article-body');
+
+    var allSeries = [
+        {
+            id: 'experience',
+            pageId: 'lifeandstyle/2016/mar/25/i-paid-to-have-my-daughter-kidnapped-experience',
+            title: 'I was attacked by a seal',
+            trail: '‘It rolled on top of me, pushing me under water. I was trapped under a mountain of blubber and muscle…’'
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js]: is this a real article?

**Comment by johnduffell** [comment]
go for it! I'd click it ;) :+1:

File: static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js (hunk 1 above)
Comment (OliverJAsh) [inline static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js]: Yup!

Comment (rich-nguyen) [inline static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js]: Agreed, i think this line is the dominant factor in whether the test is successful or not! :swimmer:

**Comment by OliverJAsh** [comment]
We will be comparing the CTR of this to the previous testing we did (without cutout). It's not a completely fair A/B test, this is more of a tactical thing for our work with editorial…

File: static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js (hunk 1 above)
Comment (OliverJAsh) [inline static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js]: It goes live tomorrow

**Comment by superfrank** [comment]
seal

**Comment by paperboyo** [comment]
Most probably not worth it, but here's ~4x smaller cutout:
https://uploads.guim.co.uk/2016/03/25/seal2.png

Having a weekly subscription to human interest stories about being saved from seal attack by a dolphin cheating on its partner - totally worth it!

**Pull request merged**